Gemfile

```diff
@@ -3,6 +3,8 @@ source 'https://rubygems.org'
 #   spec.add_runtime_dependency '<name>', [<version requirements>]
 gemspec name: 'metasploit-framework'
 
+gem 'metasploit_data_models', git: 'git@github.com:cdelafuente-r7/metasploit_data_models.git', branch: 'MS-9930_resource_layered_services'
+
```

Comment:
This has been added to make testing possible before the `metasploit_data_models` PR is landed. This will need to be removed before landing this PR